bin/run_many

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
+
+# Copyright 2023, OpenHW Group
+# SPDX-License-Identifier:Apache-2.0 WITH SHL-2.1
+
+# Dead-simple script to run a single test multiple times and report pass/fail results.
+# Uses a different seed for each run.
+# Generates the same "make test" command generated by the "ci_check" script.
+#
+# Usage:   run_many [test-program] [simulator] [nruns]
+#          For example, "run_many hello-world vcs 2" will run the hello-world
+#          test-program with VCS twice.
+#
+# TODO:    command-line argument processing is primitive.
+
+if [ $# -gt 0 ]
+then
+  TESTPROGRAM=$1
+else
+  TESTPROGRAM=hello-world
+fi
+
+if [ $# -gt 1 ]
+then
+  SIM=$2
+else
+  SIM=vcs
+fi
+
+if [ $# -gt 2 ]
+then
+  RUNS=$3
+else
+  RUNS=2
+fi
+let RUNSM1=$RUNS-1
+
+if [[ -z "${CV_CORE}" ]];
+then
+  echo CV_CORE not defined... cannot proceed.
+  exit 1
+else
+  cd ../${CV_CORE}/sim/uvmt
+  echo "Running in ../${CV_CORE}/sim/uvmt"
+fi
+
+echo "IMPORTANT!!! Do you want to skip 'make clean_all' [Y/N]?"
+read YORN
+if [ $YORN != "Y" ];
+then
+  echo "make clean_all SIMULATOR=$SIM"
+  make clean_all SIMULATOR=$SIM
+fi
+
+counter=0
+until [ $counter -gt $RUNSM1 ]
+do
+  THISSEED=`date +%N`
+  echo "make test SEED=$THISSEED TEST=$TESTPROGRAM SIMULATOR=$SIM GEN_START_INDEX=$counter RUN_INDEX=$counter USE_ISS=NO"
+  make test SEED=$THISSEED TEST=$TESTPROGRAM SIMULATOR=$SIM GEN_START_INDEX=$counter RUN_INDEX=$counter USE_ISS=NO
```

Comment:
This got me curious - I have always been using `RNDSEED` as opposed to `SEED` when running tests, it looks like the `./mk/uvmt/uvmt.mk` made this unnecessarily complex:
```
# Seed management for constrained-random sims
SEED    ?= 1
RNDSEED ?=

ifeq ($(SEED),random)
RNDSEED = $(shell date +%N)
else
ifeq ($(SEED),)
# Empty SEED variable selects 1
RNDSEED = 1
else
RNDSEED = $(SEED)
endif
```

Couldn't this just have been rewritten:
```
SEED ?= 1
ifeq ($(SEED),random)
RNDSEED = $(shell date +%N)
else
RNDSEED ?= $(SEED)
endif
```